### DISCUSSION TITLE
Test Suite

### DISCUSSION THREAD

**Discussion by duncanmcclean** [statamic-rad-pack/meilisearch#22] [Feature Requests]
This addon doesn't currently have any tests. 

It would be great if we could add some tests to this addon to cover things like `search:update`/creating/updating/deleting items from the search index.

As part of the GitHub Actions workflow, we could potentially install Mellisearch and run our tests against an actual Mellisearch instance.

**Comment by ryanmitchell** [comment]
Closing this as a test suite is now available.
